Acumatica REST API Console Application/ODataExample.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using Acumatica.RESTClient.Api;
+using Acumatica.RESTClient.Client;
+
+namespace AcumaticaRestApiExample
+{
+	public class ODataExample
+	{
+		public static void ODataGetV3(string basePath, string username, string password)
+		{
+			Console.WriteLine("OData version 3 examples");
+			var version = "OData";
+			var configuration = new Configuration(basePath);
+
+			Console.WriteLine("Testing sign in");  
+			ODataAPI odata = new ODataAPI(configuration);
+			var response = odata.Get(version, null, null, username, password);
+			Console.WriteLine(response);
+
+			Console.WriteLine("Example retrieval of customers with OData Version 3");
+			var parameter = new Dictionary<string, string>();
+			parameter.Add("format", "json");
```

Comment:
I think we should always just use json for Odata, no need to pass it from where you invoke the method